aca_weekly_report/report.py

```diff
@@ -243,7 +243,10 @@ def get_max_tccd(start, stop):
     if stop is None or DateTime(stop).secs > get_time_range("AACCCDPT")[1]:
         return np.nan
     else:
-        return np.max(fetch_sci.Msid("AACCCDPT", start, stop).vals)
+        t_ccd = fetch_sci.Msid("AACCCDPT", start, stop)
+        if t_ccd is None or len(t_ccd.vals) == 0:
```

Comment:
I don't see how `t_ccd` can be `None`. `fetch_sci.Msid()` is a class initializer that will either return a `Msid()` object or raise an exception, unless I'm very confused.

Reply:
And apparently I did fix that and the relevant commits are just hanging out on HEAD because that's where one tests this code for now.  Oops.